extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -257,17 +258,16 @@ class QueueProcessor extends EventEmitter {
             this._setupRedis(redisConfig);
         }
 
-        // FIXME support multiple scality destination sites
         if (Array.isArray(destConfig.bootstrapList)) {
-            destConfig.bootstrapList.forEach(dest => {
-                if (Array.isArray(dest.servers)) {
-                    this.destHosts =
-                        new RoundRobin(dest.servers, { defaultPort: 80 });
-                    if (dest.echo) {
-                        this._setupEcho();
-                    }
+            const dest = destConfig.bootstrapList.find(d => d.site === this.site);
```

Comment:
for this I gather that the QueueProcessor only uses a single site from `dest.bootstrapList` : so should we not instead "select" the appropriate site's bootstrap list/endpoint(s) when building `destConfig`?

(since building every destConfig is done once, when loading the config, this would also allow to do it efficiently, without going searching linearly for every site)

Reply:
done